src/libUtils/DataConversion.cpp

```diff
@@ -45,6 +45,20 @@ bool DataConversion::HexStrToUint8Vec(const string& hex_input, bytes& out) {
   return true;
 }
 
+bytes DataConversion::HexStrToUint8VecRet(const string& hex_input) {
```

Comment:
I have added these convenience functions which mirror HexStrToUint8Vec etc. except they return the thing. Usually used for debugging.